src/ngx_http_upsync_module.c

```diff
@@ -3676,6 +3676,40 @@ ngx_http_upsync_set(ngx_conf_t *cf, ngx_command_t *cmd, void *conf)
     return NGX_CONF_OK;
 }
 
+static void ngx_http_upsync_show_upstream(ngx_http_upstream_srv_conf_t *uscf, ngx_buf_t *b) {
+    ngx_http_upstream_rr_peers_t    *peers = NULL;
+    ngx_uint_t                       i;
+    ngx_str_t                       *host;
+
```

Comment:
In my style, the lines would be:
 ngx_str_t                                       *host;
 ngx_uint_t                                      i;
 ngx_http_upstream_rr_peers_t    *peers = NULL;
Any convenience, making a change will be good.